pom.xml

```diff
@@ -22,8 +22,8 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>2.14.3.Final</quarkus.version>
-        <org.jooq.version>3.16.11</org.jooq.version>
+        <quarkus.version>3.0.0.Alpha2</quarkus.version>
+        <org.jooq.version>3.17.6</org.jooq.version>
```

Comment:
IMO we should take JOOQ version upgrade in another PR.

P.S. JOOQ 3.17 requires Java 17 as minimum Java version.

Reply:
Yes, you are right [verions](https://www.jooq.org/download/versions#oss). `3.16.11` it is then